apps/foundation-demo/src/app/checkout/components/forms/address-form/factories/address-form.factory.spec.ts

```diff
@@ -26,23 +26,49 @@ describe('Foundation Demo | Checkout | Forms | Address Form | Factories | Addres
     
     let result: FormGroup;
     let expectedResult;
+    let address;
 
-    beforeEach(() => {
-      expectedResult = {
-        firstname: '',
-        lastname: '',
-        street: '',
-        city: '',
-        state: 'State',
-        postcode: '',
-        telephone: ''
-      };
-
-      result = addressFormFactory.create();
+    describe('when address is null', () => {
+      
+      beforeEach(() => {
+        address = null;
+        expectedResult = {
```

Comment:
Should use the address factory out of `@daffodil/core/testing`

Reply:
so the address created by the address factory has an email, while this one doesn't. Switching it for the address factory will break this test.

Reply:
I think the billing addresses need an email. This allows us to know about the email address of a guest customer. Otherwise, we have no idea the customer's email when they check out (unless they make an account).

Reply:
Upon thinking about this, I take it back. Addresses don't have emails. Addresses should be physical locations.